rmqsrc/decode.nim

```diff
@@ -200,11 +200,17 @@ proc decodeConnectionStart(data: Stream): Method =
 
 proc decodeMethod(data: Stream): Method =
   let
-    class = data.readClassId.Class
-    methodId = data.readMethodId.MethodId
+    classNum = data.readClassId
+    methodNum = data.readMethodId
+
+  let
+    pair = toMethod(classNum, methodNum)
```

Comment:
This can be expressed as `let (class, methodId) = ...`

Reply:
Ah good to know... I tried to do this without the parens before and it didn't work